src/components/ButtonPanel.js

```diff
@@ -1,44 +1,42 @@
-import React,{Component} from 'react';
+import React from 'react';
 import Buton from './Button';
 
-class ButtonPanel extends Component {
-    render(){
+function ButtonPanel(props){
         return (
             <div className="ButtonPanel">
                 <div className="ButtonPanel-Groups">
                     <div className="ButtonPanel-Group">
-                        <Buton name="AC" color="#e0e0e0" wide={false} />
-                        <Buton name="+/-" color="#e0e0e0" wide={false}  />
-                        <Buton name="%" color="#e0e0e0" wide={false}  />
-                        <Buton name="÷"  wide={false} />                        
+                        <Buton name="AC" color="#e0e0e0" wide={false} clickHandler={props.clickHandler} />
+                        <Buton name="+/-" color="#e0e0e0" wide={false}  clickHandler={props.clickHandler} />
+                        <Buton name="%" color="#e0e0e0" wide={false}  clickHandler={props.clickHandler} />
+                        <Buton name="÷"  wide={false}  clickHandler={props.clickHandler} />                        
                     </div>
                     <div className="ButtonPanel-Group">
-                        <Buton name="7" color="#e0e0e0" wide={false}/>
-                        <Buton name="8" color="#e0e0e0" wide={false}/>
-                        <Buton name="9" color="#e0e0e0" wide={false}/>
+                        <Buton name="7" color="#e0e0e0" wide={false} clickHandler={props.clickHandler} />
+                        <Buton name="8" color="#e0e0e0" wide={false} clickHandler={props.clickHandler} />
+                        <Buton name="9" color="#e0e0e0" wide={false} clickHandler={props.clickHandler}/>
                         <Buton name="x"  wide={false}/>
```

Comment:
The click handler for multiply is missing, so no multiplication so far.

Reply:
Done